scribble-lib/scribble/private/tag.rkt

```diff
@@ -0,0 +1,39 @@
+#lang scheme/base
+
+;; It might make sense to make these functions public, but since they weren't originally,
+;; I am going to keep them in the private folder for now.
+
+(require racket/list
+         scribble/core
+         "../tag.rkt")
+
+(provide (all-defined-out))
```

Comment:
So, I know the Racket style guide says that. And in theory its nice. But honestly, I've really given up on it. Because `contract-out` and all of the predicate contracts defined in other modules need to be required above the provide. And having two require blocks is ugly and unmaintainable.